### DISCUSSION TITLE
Is there operator for f# that works like ?? in csharp

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/fsharp#13768] [Q&A] [answered]
If there is a value ``` myVal : Option<'T> ``` that's ``` None ``` I would like to provide an alternative without writing ``` match ``` by
``` let x = myVal ?? AnotherValue ```

**Comment by En3Tho** [comment, marked as answer]
No, there is not. I tried doing something like that but C# has special handling of this operator (especially precedence).

Option.defaultValue/defaultWith is the way to go in F#.
